tests/codegen/riscv-target-abi.rs

```diff
@@ -0,0 +1,20 @@
+//@ revisions:riscv64gc riscv32gc riscv32imac
+
+//@[riscv64gc] compile-flags: --target=riscv64gc-unknown-linux-gnu
+//@[riscv64gc] needs-llvm-components: riscv
+// riscv64gc: !{i32 {{1}}, !"target-abi", !"lp64d"}
+
+//@[riscv32gc] compile-flags: --target=riscv32gc-unknown-linux-musl
+//@[riscv32gc] needs-llvm-components: riscv
+// riscv32gc: !{i32 {{1}}, !"target-abi", !"ilp32d"}
+
+//@[riscv32imac] compile-flags: --target=riscv32imac-unknown-none-elf
+//@[riscv32imac] needs-llvm-components: riscv
+// riscv32imac-NOT: !{i32 {{1}}, !"target-abi", !""}
```

Comment:
```suggestion
// riscv32imac-NOT: !"target-abi"
```
I'd be more liberal with the CHECK-NOT, otherwise there is a risk it will not trigger due to some minor difference in the output.

Reply:
Fixed